### PR TITLE
Movie details feature

### PR DESCRIPTION
**Pull request by lkkolosowski**
Implemented movie details feature 😊🎉

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
+import MovieDetails from "./features/MovieDetails";
 
 function App() {
   return (
     <HashRouter>
       <Header />
       <Switch>
-        <Route path="/movies">
+        <Route path={toMovie()}>
+          <MovieDetails />
```
Comment (Albrecht-Albi) [inline src/App.js:14]: You imported the components well, I was just thinking about them ;)

File: src/common/Header/styled.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { ReactComponent as VideoIcon } from "../../icons/video.svg";
 import { ReactComponent as LoupeIcon } from "../../icons/loupe.svg";
 
 export const Wrapper = styled.header`
-  background-color: ${({ theme }) => theme.color.woodsmoke};
+  background-color: ${({ theme }) => theme.color.black};
```
Comment (Albrecht-Albi) [inline src/common/Header/styled.js:6]: This color looks great

File: src/common/Loader/styled.js [hunk 3]
```diff
@@ -4,6 +4,16 @@ import { ReactComponent as LoaderIcon } from "../../icons/loader.svg";
...
+  opacity: 0;
+
+  animation: reveal 0.5s ease-in-out 0.5s;
+  animation-fill-mode: forwards;
+
+  @keyframes reveal {
+    100% {
+      opacity: 1;
+    }
+  }
```
Comment (Albrecht-Albi) [inline src/common/Loader/styled.js:16]: Cool animation code

File: src/common/MainWrapper/index.js [hunk 4]
```diff
@@ -1,10 +1,9 @@
...
-    </PageWrapper>
-);
\ No newline at end of file
+export const MainWrapper = ({ content }) => (
+  <PageWrapper>{content}</PageWrapper>
+);
+
+export const MainBackdropWrapper = ({ content }) => (
+  <PageWrapper backdrop>{content}</PageWrapper>
+);
```
Comment (Albrecht-Albi) [inline src/common/MainWrapper/index.js]: you imported well title

File: src/common/Tiles/Genres/styled.js [hunk 5]
```diff
@@ -15,9 +20,23 @@ export const Genre = styled.p`
...
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+    ${({ details }) =>
+      details &&
+      css`
+        align-self: flex-start;
+        font-size: 10px;
+        padding: 4px 8px;
+      `}
```
Comment (Albrecht-Albi) [inline src/common/Tiles/Genres/styled.js:40]: Responsive looks better now, good move

File: src/common/Tiles/Images/styled.js [hunk 6]
```diff
@@ -10,8 +10,26 @@ export const Wrapper = styled.div`
...
+      }
+    `}
+
   @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
-    margin-right: 16px;
+    ${({ movieDetails }) =>
+      !movieDetails &&
+      css`
+        margin-right: 16px;
+      `}
```
Comment (Albrecht-Albi) [inline src/common/Tiles/Images/styled.js:32]: Responsive looks better now, good move :)

File: src/common/Tiles/Info/index.js [hunk 7]
```diff
@@ -1,9 +1,46 @@
...
+    )}
+  </Wrapper>
+);
+
+export const MainInfoDetails = ({ title, year }) => (
+  <>
+    <TitleDetails>{title}</TitleDetails>
+    {year && <SubtitleDetails>{extractFirstWord(year)}</SubtitleDetails>}
+  </>
+);
```
Comment (Albrecht-Albi) [inline src/common/Tiles/Info/index.js:46]: You've created an interesting component

File: src/features/MovieDetails/index.js [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+            production={details.production_countries}
+            release={details.release_date}
+          />
+        }
+      />
+    </>
+  );
+};
+
+export default MovieDetails;
```
Comment (Albrecht-Albi) [inline src/features/MovieDetails/index.js:75]: I can see that you put a lot of work into this component, bravo, I like it

File: src/features/MovieDetails/styled.js [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+export const BackdropContent = styled.div`
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  margin-bottom: 56px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+    margin-bottom: 8px;
+  }
```
Comment (Albrecht-Albi) [inline src/features/MovieDetails/styled.js:67]: very nice styling

**Review by Albrecht-Albi — APPROVED** [review]

File: src/common/Loader/styled.js (hunk 3 above)
Comment (lkkolosowski) [reply src/common/Loader/styled.js:16]: Im glad you like it

File: src/common/Tiles/Images/styled.js (hunk 6 above)
Comment (lkkolosowski) [reply src/common/Tiles/Images/styled.js:32]: there is always something to improve haha 😁

**Pull request merged**